pkg/cluster/graph/manager.go

```diff
@@ -42,33 +41,31 @@ func NewManager(log *logrus.Entry, aead encryption.AEAD, storage storage.Manager
 func (m *manager) Exists(ctx context.Context, resourceGroup, account string) (bool, error) {
 	m.log.Print("checking if graph exists")
 
-	blobService, err := m.storage.BlobService(ctx, resourceGroup, account, mgmtstorage.Permissions("r"), mgmtstorage.SignedResourceTypesO)
+	blobService, err := m.storage.BlobService(ctx, resourceGroup, account, armstorage.Permissions("r"), armstorage.SignedResourceTypesO)
 	if err != nil {
 		return false, err
 	}
 
-	aro := blobService.GetContainerReference("aro")
-	return aro.GetBlobReference("graph").Exists()
+	return blobService.BlobExists(ctx, "aro", "graph")
```

Comment:
`aro` is the container, `graph` is the blob?  

Are we confirming the existance of both?

Reply:
Yes, it will throw the error if either of them doesn't exist.

Reply:
I see.  It's `BlobExists(ctx, container, blobname)`.  The magic strings moving will help here.